### ISSUE TITLE
[Feature] Providing an option for choosing Custom section Type

### ISSUE THREAD

**Issue by technodefacto** [AmruthPillai/Reactive-Resume#1987]
### Is there an existing issue for this feature?

- [X] Yes, I have searched the existing issues and it doesn't exist.

### Feature Description

# Feature Description

## Current: 
After adding a new section, the next step is to add a new item where you have a item template to fill in your details. Currently that template is similar to "_project section_" see the **Image1** below. But, if one wants to have this item template to be like "_skills section_" see **Image2** or any other "item template" available within the section interface it is not possible now. 
**Image1:**
![image](https://github.com/AmruthPillai/Reactive-Resume/assets/58810077/6ae2c46e-ab3b-463c-8c3a-01a9601c8475)
**Image2:**
![image](https://github.com/AmruthPillai/Reactive-Resume/assets/58810077/01445668-2efb-4f5c-88b8-cd6a14ea4f01)

## Required Changes:
It would be nice if a person can choose the "item template" for his newly created section. 

Thank you for this wonderful tool.

**Comment by TAYALE** [comment]
I agree. Language is separate from skills. But I also want to differ soft and hard skills which is not possible now.

**Comment by PeatGerman** [comment]
I agree with you, I hope it will be implemented quickly

**Comment by amruthpillai** [comment]
Closing this issue as I'm doing some clean up to move to a newer version of the project. I apologize in advance for the inconvenience if you find the need to reopen this issue.

Please read #2499 for more information about the migration.
